inst/@sym/assumeAlso.m

```diff
@@ -30,15 +30,7 @@
 %% @group
 %% syms x integer
 %% x1 = x;
-%% assumptions(x1)
-%%   @result{} ans =
-%%     @{
-%%       [1,1] = x: integer
-%%     @}
```

Comment:
Why to remove this doctest? Removing the `+SKIP` is ok but why this test?
Am I missing something!?